src/openrct2/scripting/bindings/world/ScTileElement.cpp

```diff
@@ -2049,6 +2062,8 @@ namespace OpenRCT2::Scripting
         dukglue_register_property(ctx, &ScTileElement::primaryColour_get, &ScTileElement::primaryColour_set, "primaryColour");
         dukglue_register_property(
             ctx, &ScTileElement::secondaryColour_get, &ScTileElement::secondaryColour_set, "secondaryColour");
+        dukglue_register_property(
```

Comment:
amusing that the comment was still correct just the code was missing

Reply:
Not really. The comment only refered to 1st and 2nd colour. 3rd colour was handled below since it previously was for only walls. So since 3rd colour is now implemented for both LS and SS, I just moved this call closer to the other colour register calls.